src/mumble/MainWindow.cpp

```diff
@@ -564,6 +562,14 @@ void MainWindow::updateTransmitModeComboBox() {
 	}
 }
 
+QMenu *MainWindow::createPopupMenu() {
+	if (g.s.wlWindowLayout == Settings::LayoutCustom) {
+		return QMainWindow::createPopupMenu();
+	}
+
+	return new QMenu();
```

Comment:
Wow. How did I miss that. You are right. I'd still return ```NULL``` instead of an empty menu as that might make a difference.